src/Tools/Source/CompilerGeneratorTools/Source/CSharpErrorFactsGenerator/CSharpErrorFactsGenerator.csproj

```diff
@@ -2,13 +2,24 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Platform Condition="'$(Platform)' == ''">x64</Platform>
-    <PlatformTarget>x64</PlatformTarget>
-    <Platforms>x64</Platforms>
-    <OutputType>Exe</OutputType>
+    <Platform Condition="'$(Platform)' == ''">AnyCPU</Platform>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <Platforms>AnyCPU</Platforms>
     <RootNamespace>Roslyn.Compilers.CSharp.Internal.CSharpErrorFactsGenerator</RootNamespace>
     <AutoGenerateBindingRedirects>True</AutoGenerateBindingRedirects>
-    <TargetFramework>net7.0</TargetFramework>
+    <!-- We build this against net7.0 to build the console app version, and against netstandard2.0 for
+         the Source Generator version. -->
+    <TargetFrameworks>net7.0;netstandard2.0</TargetFrameworks>
```

Comment:
nit: should we do `netcoreapp3.1;netstandard2.0` as we do for the syntax generator? Keeping the versions closer could help avoid mistakes (using new APIs unavailable in standard)

Reply:
@jcouv The syntax generator is net7.0. This is exactly a copy from there:

https://github.com/dotnet/roslyn/blob/bfae7f98d60d9eb99a6b9053c0a4d2c7a304afa4/src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/CSharpSyntaxGenerator.csproj#L10-L12

Reply:
Thanks for the clarification.
I was looking at the commit you had linked above, but it looks like this was changed since.